Rakefile

```diff
@@ -1,22 +1,6 @@
-#!/usr/bin/env rake
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
 
-begin
-  require 'bundler/setup'
-rescue LoadError
-  puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
+RSpec::Core::RakeTask.new(:spec) do |task|
+  task.rspec_opts = ['--color', '--format', 'documentation']
 end
```

Comment:
@Dantemss - I believe they are NOT necessary - everything seems to be working just fine.  Unless you have a specific example that needs inclusion, I'm going to consider this closed.